backend/src/repositories/PoolsRepository.ts

```diff
@@ -93,6 +93,11 @@ class PoolsRepository {
       const [rows] = await connection.query(query, [slug]);
       connection.release();
 
+      if (rows.length < 1) {
+        logger.debug(`$getPool(): slug ${slug} does not match any known pool`);
```

Comment:
I believe it could be a risk to print and broadcast unfiltered data like this. So either not print out slug, or filter it somehow first.

Reply:
I've escaped it using the `connection` object. See https://github.com/mempool/mempool/pull/1487/commits/1107c0e13a5f4dfb96ae09e8e52888199400dbb2#diff-9d0747e0a073ef0a7fb8bc981126d3eeebf2c207566bade3f8e8fa1ba7fffddfR97